README.md

```diff
@@ -1,4 +1,4 @@
-# Chainlink [![Travis-CI](https://travis-ci.org/smartcontractkit/chainlink.svg?branch=master)](https://travis-ci.org/smartcontractkit/chainlink) [![Maintainability](https://api.codeclimate.com/v1/badges/273722bb9f6f22d799bd/maintainability)](https://codeclimate.com/github/smartcontractkit/chainlink/maintainability) [![Go Report Card](https://goreportcard.com/badge/github.com/smartcontractkit/chainlink)](https://goreportcard.com/report/github.com/smartcontractkit/chainlink) [![Join the chat at https://gitter.im/smartcontractkit-chainlink/Lobby](https://badges.gitter.im/smartcontractkit-chainlink/Lobby.svg)](https://gitter.im/smartcontractkit-chainlink/Lobby) [![codecov](https://codecov.io/gh/smartcontractkit/chainlink/branch/master/graph/badge.svg)](https://codecov.io/gh/smartcontractkit/chainlink) [![GoDoc](https://godoc.org/github.com/smartcontractkit/chainlink?status.svg)](https://godoc.org/github.com/smartcontractkit/chainlink)
+# Chainlink [![Travis-CI](https://travis-ci.org/smartcontractkit/chainlink.svg?branch=master)](https://travis-ci.org/smartcontractkit/chainlink) [![Maintainability](https://api.codeclimate.com/v1/badges/273722bb9f6f22d799bd/maintainability)](https://codeclimate.com/github/smartcontractkit/chainlink/maintainability) [![Go Report Card](https://goreportcard.com/badge/github.com/smartcontractkit/chainlink)](https://goreportcard.com/report/github.com/smartcontractkit/chainlink) [![Join the chat at https://gitter.im/smartcontractkit-chainlink/Lobby]](https://badges.gitter.im/smartcontractkit-chainlink/Lobby.svg)](https://gitter.im/smartcontractkit-chainlink/Lobby) [![codecov](https://codecov.io/gh/smartcontractkit/chainlink/branch/master/graph/badge.svg)](https://codecov.io/gh/smartcontractkit/chainlink) [![GoDoc](https://godoc.org/github.com/smartcontractkit/chainlink?status.svg)](https://godoc.org/github.com/smartcontractkit/chainlink)
```

Comment:
Applying this change breaks in markdown preview locally.
To explain, the Gitter tag has a URL as part of the initial `![text]` tag, while the others do not, so it can seem like it needs one. So that initial tag should look like `![Join the chat at https://gitter.im/smartcontractkit-chainlink/Lobby]` instead of adding another `]` because there's 3 URLs on this line.

Reply:
Ah this might be some bad behavior on behalf of vim's syntax highlighting for markdown. Will close.